jupyterhub/tests/mockservice.py

```diff
@@ -1,44 +1,49 @@
-"""Mock service for testing
+"""Helpers for testing a Service
```

Comment:
This script *is* a mock service for testing, it does not provide helpers. It has the main function at the bottom. It is a complete application to be started by the `mockservice` fixtures in `conf.py`.

How about this docstring:

> Mock service for testing Service integration
>
> A JupyterHub service running a basic HTTP server.
> Used by the mockservice fixtures.
>
> Handlers allow:
> - echoing proxied URLs back
> - retrieving service's environment variables
> - testing service's API access to the Hub